sencha-workspace/SlateDemonstrationsStudent/app/view/Dashboard.js

```diff
@@ -221,6 +246,16 @@ Ext.define('SlateDemonstrationsStudent.view.Dashboard', {
         return Ext.factory(cardsCt, 'SlateDemonstrationsStudent.view.CardsContainer', oldCardsCt);
     },
 
+    applyNonEnrollmentMessage: function(message, oldMessage) {
+        if (typeof message === 'boolean') {
+            message = {
+                hidden: !message
+            };
+        }
+
+        return Ext.factory(message, 'SlateDemonstrationsStudent.view.NonEnrollmentMessage', oldMessage);
```

Comment:
@BillClinton this PR doesn't appear to be adding a new file for this new class `SlateDemonstrationsStudent.view.NonEnrollmentMessage` so this won't work

Reply:
This just worked, maybe because the second 2 arguments are optional.   But I should have looked into the factory method anyway and just sent the first argument if it is working that way